scripts/imgtool/main.py

```diff
@@ -205,16 +205,18 @@ def convert(self, value, param, ctx):
 @click.option('--align', type=click.Choice(['1', '2', '4', '8']),
               required=True)
 @click.option('-k', '--key', metavar='filename')
+@click.option('-L', '--load-addr', type=BasedIntParamType(), required=False,
```

Comment:
Could you move this option up? `click` sorts them in reverse (bottom to top) and I would rather see the `required` ones first in the help page.